source/whats-new.txt

```diff
@@ -37,6 +37,9 @@ What's New in 1.9
   properties. You should use ``startup.mode=copy_existing`` and
   ``startup.mode.copy.existing.*`` properties to configure the copy existing
   feature.
+- Introduced ``change.stream.full.document.before.change`` setting that
+  allows you to access and configure the pre-image of an update
```

Comment:
Issue:

Grammar error. I believe this should be either something like "Introduced the...setting" or "Introduced a...setting" or drop the word "setting" to get "Introduced ``change.stream.full.document.before.change`` which enables you to..."